pages/examples/animations/card-hand.tsx

```diff
@@ -43,7 +43,7 @@ function ExampleCardHand(props) {
       >
         <CardHandLayout cards={cards} expanded={expanded} onExpand={() => setExpanded(true)} />
       </GridLayout>
-      <GlobalModalManager viewer={props.viewer} />
+      <GlobalModalManager />
```

Comment:
I see how this is better in some cases, I would almost name it as `GlobalModalManagerWithViewerContext`, but I need to think about this more on what the most obvious name is, I just don't want the question of "How does this thing get the authenticated user? Oh theres a context somewhere..."

Reply:
this could be fixed by the proposition in my other comment.